docker/README_cn.md

```diff
@@ -4,61 +4,125 @@
 
-		$ docker run -t --rm -p 8501:8501 --device /dev/dri:/dev/dri -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=GPU openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
+此处为Resnet50模型使用OpenVINO Integration with Tensorflow实例，提供了REST API相关客户端脚本。
 
-	在Myriad上运行：
+1. 从TF社区下载[Resnet50 model](#https://storage.googleapis.com/tfhub-modules/google/imagenet/resnet_v2_50/classification/5.tar.gz)并将其目录解压至`resnet_v2_50_classifiation/5`文件夹。 
 
-		$ docker run -t --rm -p 8501:8501 --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=MYRIAD openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
+2. 启动resnet50模型的服务容器：
+	
+	在**CPU**后端上运行：
+
+		docker run -it --rm \
+			   -p 8501:8501 \
+			   -v <path to resnet_v2_50_classifiation>:/models/resnet \
+			   -e MODEL_NAME=resnet \
+			   openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving
+
+	在**iGPU**上运行：
+
+		docker run -it --rm \
+			   -p 8501:8501 \
+			   --device-cgroup-rule='c 189:* rmw' \
+			   --device /dev/dri:/dev/dri \
+			   -v <path to resnet_v2_50_classifiation>:/models/resnet \
+			   -e MODEL_NAME=resnet \
+			   -e OPENVINO_TF_BACKEND=GPU \
+			   openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving
+
+	在**MYRIAD**上运行：
+
+		docker run -it --rm \
+			   -p 8501:8501 \
+			   --device-cgroup-rule='c 189:* rmw' \
+			   -v /dev/bus/usb:/dev/bus/usb \
+			   -v <path to resnet_v2_50_classifiation>:/models/resnet \
+			   -e MODEL_NAME=resnet \
+			   -e OPENVINO_TF_BACKEND=MYRIAD \
+			   openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving
+	
+	在**MYRIAD**上运行：
+
+		docker run -itu root:root --rm \
+			   -p 8501:8501 \
+			   --device-cgroup-rule='c 189:* rmw' \
+			   -v /dev/bus/usb:/dev/bus/usb \
+			   --mount type=bind,source=/var/tmp,destination=/var/tmp \
+			   --device /dev/ion:/dev/ion \
+			   -v <path to resnet_v2_50_classifiation>:/models/resnet \
+			   -e MODEL_NAME=resnet \
+			   openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving
 
 3. 运行脚本从客户端发送推理请求并从服务器获取预测。
-
-		$ wget https://raw.githubusercontent.com/tensorflow/serving/master/tensorflow_serving/example/resnet_client.py
-		$ python resnet_client.py
+		wget https://raw.githubusercontent.com/tensorflow/serving/master/tensorflow_serving/example/resnet_client.py
+		python resnet_client.py
 
 # 预构建镜像
 
-- [Docker* Hub](https://hub.docker.com/u/openvino/)
-- [Azure* Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/intel_corporation.openvinotensorflow)
+- [Ubuntu18 runtime image on Docker* Hub](https://hub.docker.com/r/openvino/openvino_tensorflow_ubuntu18_runtime)
+- [Ubuntu20 runtime image on Docker* Hub](https://hub.docker.com/r/openvino/openvino_tensorflow_ubuntu20_runtime)
+- [Azure* Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/intel_corporation.openvino_tensorflow)
```

Comment:
Underscore needs to be remove from Azure links

Reply:
Updated